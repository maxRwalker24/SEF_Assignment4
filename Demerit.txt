56s_d%&fAB 10-09-2024 4
56s_d%&fAB 10-10-2024 5
56s_d%&fAB 10-11-2024 5
