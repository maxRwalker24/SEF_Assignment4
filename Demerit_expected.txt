56s_d%&fAB 14-01-2024 1
56s_d%&fAB 23-05-2025 3
56s_d%&fAB 10-09-2005 6